_config.yml

```diff
@@ -86,12 +87,15 @@ just_the_docs:
       nav_exclude: false
       search_exclude: false
 
+markdown: kramdown
+
 # Kramdown settings
 kramdown:
   parse_block_html: true
+  syntax_highlighter: rouge
   syntax_highlighter_opts:
     block:
-      line_numbers: true
+      line_numbers: false
```

Comment:
Interesting. Line numbers are stilling showing up for me (eg. apache.md)

Reply:
OH! I see the linenos tag is added in places where it's needed! That's so much better.